lib/routes/podcast/payment_adjustment.dart

```diff
@@ -255,8 +257,11 @@ class PaymentAdjustmentState extends State<PaymentAdjustment> {
                           key: boostWidgetKey,
                           userModel: userModel,
                           onBoost: (int boostAmount, {String boostMessage}) {
-                            paymentsBloc.actionsSink.add(PayBoost(boostAmount,
-                                boostMessage: boostMessage));
+                            paymentsBloc.actionsSink.add(PayBoost(
+                              boostAmount,
+                              boostMessage: boostMessage,
+                              senderName: _senderName,
```

Comment:
Since we already listen to the user stream can we take the send name from `userModel.name`  and remove the `_senderName` property?